lang/po/new/cs.po

```diff
@@ -783,7 +780,7 @@ msgstr "MK3 firmware detekovan na MK3S tiskarne"
 # MSG_NORMAL c=7
 #: messages.c:107
 msgid "Normal"
-msgstr ""
+msgstr "Normal"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.